packages/context/README.md

```diff
@@ -321,43 +322,50 @@ following data:
 -   `type`
     -   The type of update. Possible values are `BlobCreated`, `BlobDeleted`, and `BlobMetadataUpdated`.
 
-##### `documentChanges.unsubscribe(objectId, instanceId, (data: DocumentChange[]) => {})`
+##### `documentChanges.unsubscribe(objectId, instanceId, (changes: DocumentChange[]) => {})`
 
-Unsubscribe to the specified object instance document changes.
+Unsubscribe to the specified object instance changes.
 
 Callback function is optional.
+If callback function is not defined, nothing will not be invoked when a connection is closed.
```

Comment:
You've got a double negative "nothing will not be invoked", and I'm not exactly sure what this is saying, but it sounds like this is saying it will be a no-op.  I don't think that's the case, I'm not sure, but from the SignalR documentation it sounded like it would unsubscribe everything.